### ISSUE TITLE
ppf-contracts: evaluate makerdao's medianizer as potential source for ETH <> USD conversion

### ISSUE THREAD

**Issue by sohkai** [aragon/ppf#15]
See https://etherscan.io/address/0x729D19f657BD0614b4985Cf1D82531c67569197B#readContract.

We can just do a `medianizer.peek()` to get the current price of ETH in USD (and assume this is DAI).

**Comment by sohkai** [comment]
Another idea is to build a separate feed that just implements the functionality to get ETH <> USD.

**Comment by luisivan** [comment]
I think this is a great idea

**Comment by izqui** [comment]
As long as it supports the `IFeed` interface and we use the same 'fake address' for USD so it stays compatible with PPF I think it is a good idea.

> and assume this is DAI
I am not super sure about this, as Dai could lose its peg

**Comment by sohkai** [comment]
> I am not super sure about this, as Dai could lose its peg

This is true; I think we could add an operator setting to disable this feed in the future, whenever multi-collateral DAI is deployed.

> same 'fake address' for USD

What is this address BTW? We should probably document this.

**Comment by izqui** [comment]
@sohkai It is documented in the readme: https://github.com/aragon/ppf#tickers-and-token-addresses

**Comment by sohkai** [comment]
Notes from an offline discussion on this:

- We should choose what happens when the price feed is disabled
    - Revert
    - Cache last updated price and stop updating
    - Who should control this price feed?
        - Most likely AA
- Does MakerDAO assume that DAI<>USD is 1:1 in their own price feed?
- Should we also include USDC / GUSD / etc?
    - Check if people are actually using them and if it makes sense to also peg them as 1:1 to USD
